### ISSUE TITLE
Example for python usage ?

### ISSUE THREAD

**Issue by hubgitname** [txomon/ubus#3]
Hi,
Do you have a python example for a server/client working with the ubus library ?

Thanks.

**Comment by txomon** [comment]
Nope, when creating this, I realised the problem on having to attach the python runtime to the loop, and at that time python asyncio wasn't really a thing. It was too much work for the use I was going to make out of it.
Doing it in python3.6 should be doable now though.
